autogpt/core/llm/base.py

```diff
@@ -3,12 +3,11 @@
 from dataclasses import dataclass, field
 from typing import Dict, List
 
-if typing.TYPE_CHECKING:
-    from autogpt.core.configuration.base import Configuration
-    from autogpt.core.logging.base import Logger
-    from autogpt.core.planning.base import ModelPrompt
-    from autogpt.core.plugin.base import PluginManager
-    from autogpt.core.workspace.base import Workspace
+from autogpt.core.configuration.base import Configuration
```

Comment:
Don't drop the typechecking blocks yet.  There **will** be cyclic dependencies in the method signatures, but I'm still ironing out the details.  I don't want to have to come monkey with imports all the time.  Easier to just resolve this stuff with interfaces have stabilized